### ISSUE TITLE
2.4.0 | "Unexpected token 'Œâ”€â”€â”€â”€â”€â”�${e}[32mâ”Œâ”€â”€â”€â”€â”€â”�"' in expression or statement." etc.

### ISSUE THREAD

**Issue by tordenflesk** [kiedtl/winfetch#134]
```
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:462 char:33
+ ...     "${e}[${t};31mâ”Œâ”€â”€â”€â”€â”€â”${e}[32mâ”Œâ”€â”€â”€â”€â”€â”"
+                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unexpected token 'Œâ”€â”€â”€â”€â”€â”${e}[32mâ”Œâ”€â”€â”€â”€â”€â”"' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:463 char:33
+                 "${e}[${t};31mâ”‚     â”‚${e}[32mâ”‚     â”‚"
+                                 ~~~~~~~~~
Unexpected token '‚     â”‚' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:463 char:42
+                 "${e}[${t};31mâ”‚     â”‚${e}[32mâ”‚     â”‚"
+                                          ~~~~
Unexpected token '${e}' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:463 char:47
+                 "${e}[${t};31mâ”‚     â”‚${e}[32mâ”‚     â”‚"
+                                               ~
Array index expression is missing or not valid.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:463 char:47
+                 "${e}[${t};31mâ”‚     â”‚${e}[32mâ”‚     â”‚"
+                                               ~~~~~~~~~~~~~~~
Unexpected token '32mâ”‚     â”‚"
                "${e}[${t};31mâ”‚' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:468 char:33
+                 "${e}[${t};34mâ”‚     â”‚${e}[33mâ”‚     â”‚"
+                                 ~~~~~~~~~
Unexpected token '‚     â”‚' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:468 char:42
+                 "${e}[${t};34mâ”‚     â”‚${e}[33mâ”‚     â”‚"
+                                          ~~~~
Unexpected token '${e}' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:468 char:47
+                 "${e}[${t};34mâ”‚     â”‚${e}[33mâ”‚     â”‚"
+                                               ~
Array index expression is missing or not valid.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:468 char:47
+                 "${e}[${t};34mâ”‚     â”‚${e}[33mâ”‚     â”‚"
+                                               ~~~~~~~~~~~~~~~
Unexpected token '33mâ”‚     â”‚"
                "${e}[${t};34mâ””â”€â”€â”€â”€â”€â”˜${e}[33mâ””â”€â”€â”€â”€â”€â”˜"
            )
        } else {
            Write-Error 'The' in expression or statement.
At C:\Users\Tordenflesk\scoop\apps\winfetch\current\winfetch.ps1:490 char:30
+             content = ('{0}[0;40m{1}{0}[0;41m{1}{0}[0;42m{1}{0}[0;43m ...
+                              ~
Missing ']' after array index expression.
Not all parse errors were reported.  Correct the reported errors and try again.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

```
2.3.0 works fine.

**Comment by rashil2000** [comment]
Try these steps here and restart the computer:

https://www.windowsphoneinfo.com/threads/some-unicode-characters-suddenly-not-displaying-properly-in-every-program-windows-10.557285/#post-2619733

![image](https://user-images.githubusercontent.com/46838874/155540300-6eac1ccc-358d-44d9-ad22-9c00d72a0c6a.png)

**Comment by tordenflesk** [comment]
Nope:

https://i.imgur.com/aF2oyck.png

**Comment by astelmachonak-nydig** [comment]
Have similar issue with 2.4.0:
![image](https://user-images.githubusercontent.com/100310178/166283783-6de3e951-b162-4b16-acad-c0e279d3e861.png)

**Comment by rashil2000** [comment]
There is some discussion related to this in https://github.com/kiedtl/winfetch/issues/132

Do you have any suggestions for pure ASCII small Microsoft logo?

**Comment by astelmachonak-nydig** [comment]
For me the easiest fix/workaround for Powershell 5.1 was to save `winfetch.ps1` in UTF-8 with BOM instead of just UTF-8.

**Comment by tordenflesk** [comment]
Yes, UTF-8 BOM works:
https://i.imgur.com/1OZis9V.png

**Comment by rashil2000** [comment]
Okay.

@astelmachonak-nydig would you like to make a PR?

**Comment by astelmachonak-nydig** [comment]
Yes, I will look into it tomorrow.

**Comment by ryouze** [comment]
> For me the easiest fix/workaround for Powershell 5.1 was to save `winfetch.ps1` in UTF-8 with BOM instead of just UTF-8:

Works for me as well, no need to enable `Beta: Use Unicode UTF-8 for worldwide language support`.

I opened the script with Notepad++, set the encoding to `UTF-8 BOM` and then saved it.

![encoding](https://user-images.githubusercontent.com/98982999/166525250-e42bf9c5-9968-4da2-8ba8-b7d71422b0cc.jpg)

**Comment by astelmachonak-nydig** [comment]
@rashil2000 here you go https://github.com/kiedtl/winfetch/pull/135
